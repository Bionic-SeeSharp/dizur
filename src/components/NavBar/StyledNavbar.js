import styled from 'styled-components';
import { Navbar } from 'reactstrap';

const StyledNavbar = styled(Navbar)`
  height: 79px;
  background: #000000;
`;

export default StyledNavbar;
