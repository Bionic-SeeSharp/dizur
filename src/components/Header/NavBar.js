import React, { Component } from 'react';
import { NavbarToggler, Collapse } from 'reactstrap';
import StyledNav from './StyledNav';
import StyledNavbar from './StyledNavbar';
import StyledNavItem from './StyledNavItem';
import StyledNavLink from './StyledNavLink';
import StyledNavbarBrand from './StyledNavbarBrand';
import Separator from './Separator';

class NavBar extends Component {
  constructor(props) {
    super(props);
    this.toggle = this.toggle.bind(this);
    this.state = {
      isOpen: false
    };
  }
  toggle() {
    let isOpenState = this.state.isOpen;
    this.setState({ isOpen: !isOpenState });
  }

  render() {
    return <div>
        <StyledNavbar expand="md">
          <StyledNavbarBrand href="/">Spotify</StyledNavbarBrand>
          <NavbarToggler onClick={this.toggle} />
          <Collapse isOpen={this.state.isOpen} navbar>
            <StyledNav className="ml-auto" navbar>
              <StyledNavItem>
                <StyledNavLink href="/components/">Premium</StyledNavLink>
              </StyledNavItem>
              <StyledNavItem>
                <StyledNavLink href="/components/">Help</StyledNavLink>
              </StyledNavItem>
              <StyledNavItem>
                <StyledNavLink href="/components/">Download</StyledNavLink>
              </StyledNavItem>
              <StyledNavItem>
                <Separator>|</Separator>
              </StyledNavItem>
              <StyledNavItem>
                <StyledNavLink href="/components/">Sign up</StyledNavLink>
              </StyledNavItem>
              <StyledNavItem>
                <StyledNavLink href="https://github.com/reactstrap/reactstrap">
                  Log In
                </StyledNavLink>
              </StyledNavItem>
            </StyledNav>
          </Collapse>
        </StyledNavbar>
      </div>;
  }
}

export default NavBar;
