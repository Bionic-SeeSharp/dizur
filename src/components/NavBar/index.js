import React, { Component } from 'react';
import { NavbarToggler, Collapse } from 'reactstrap';
import StyledNav from './StyledNav';
import StyledNavbar from './StyledNavbar';
import StyledNavItem from './StyledNavItem';
import SignupText from './SignupText';
import Logo from '../Logo';

class NavBar extends Component {
  constructor(props) {
    super(props);
    this.toggle = this.toggle.bind(this);
    this.state = {
      isOpen: false
    };
  }
  toggle() {
    let isOpenState = this.state.isOpen;
    this.setState({ isOpen: !isOpenState });
  }

  render() {
    return (
      <div>
        <StyledNavbar expand="md" className="fixed-top">
          <Logo />
          <NavbarToggler onClick={this.toggle} />
          <Collapse isOpen={this.state.isOpen} navbar>
            <StyledNav className="ml-auto" navbar>
              <StyledNavItem>
                <SignupText>Listen</SignupText>
              </StyledNavItem>
              <StyledNavItem>
                <SignupText>|</SignupText>
              </StyledNavItem>
              <StyledNavItem>
                <SignupText>About</SignupText>
              </StyledNavItem>
            </StyledNav>
          </Collapse>
        </StyledNavbar>
      </div>
    );
  }
}

export default NavBar;
