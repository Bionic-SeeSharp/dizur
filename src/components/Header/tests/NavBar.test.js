import React from 'react';
import { shallow, mount } from 'enzyme';
import sinon from 'sinon';

import NavBar from '../NavBar';

describe('<StyledNavbar />', () => {
  it('it should make a reactstrap Navbar', () => {
    const wrapper = shallow(<NavBar />);
    expect(wrapper).toMatchSnapshot();
  });
  it('render a div', () => {
    const renderedComponent = shallow(<NavBar />);
    expect(renderedComponent.type()).toEqual('div');
  });
  it('should call the toggle method', () => {
    sinon.spy(NavBar.prototype, 'setState');
    const renderedComponent = shallow(<NavBar />);
    renderedComponent.instance().toggle();
    expect(NavBar.prototype.setState.calledOnce).toEqual(true);
  });
});