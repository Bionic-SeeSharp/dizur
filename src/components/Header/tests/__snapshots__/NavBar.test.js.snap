// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StyledNavbar /> it should make a reactstrap Navbar 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NavBar />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <Styled(Navbar)
        expand="md"
>
        <Styled(NavbarBrand)
                href="/"
        >
                Spotify
        </Styled(NavbarBrand)>
        <NavbarToggler
                onClick={[Function]}
                tag="button"
                type="button"
        />
        <Collapse
                appear={false}
                enter={true}
                exit={true}
                in={false}
                isOpen={false}
                mountOnEnter={false}
                navbar={true}
                onEnter={[Function]}
                onEntered={[Function]}
                onEntering={[Function]}
                onExit={[Function]}
                onExited={[Function]}
                onExiting={[Function]}
                tag="div"
                timeout={350}
                unmountOnExit={false}
        >
                <Styled(Nav)
                        className="ml-auto"
                        navbar={true}
                >
                        <Styled(NavItem)>
                                <Styled(NavLink)
                                        href="/components/"
                                >
                                        Premium
                                </Styled(NavLink)>
                        </Styled(NavItem)>
                        <Styled(NavItem)>
                                <Styled(NavLink)
                                        href="/components/"
                                >
                                        Help
                                </Styled(NavLink)>
                        </Styled(NavItem)>
                        <Styled(NavItem)>
                                <Styled(NavLink)
                                        href="/components/"
                                >
                                        Download
                                </Styled(NavLink)>
                        </Styled(NavItem)>
                        <Styled(NavItem)>
                                <Styled(NavLink)
                                        href="/components/"
                                >
                                        Sign up
                                </Styled(NavLink)>
                        </Styled(NavItem)>
                        <Styled(NavItem)>
                                <Styled(NavLink)
                                        href="https://github.com/reactstrap/reactstrap"
                                >
                                        Log In
                                </Styled(NavLink)>
                        </Styled(NavItem)>
                </Styled(Nav)>
        </Collapse>
</Styled(Navbar)>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <Styled(NavbarBrand)
            href="/"
>
            Spotify
</Styled(NavbarBrand)>,
          <NavbarToggler
            onClick={[Function]}
            tag="button"
            type="button"
/>,
          <Collapse
            appear={false}
            enter={true}
            exit={true}
            in={false}
            isOpen={false}
            mountOnEnter={false}
            navbar={true}
            onEnter={[Function]}
            onEntered={[Function]}
            onEntering={[Function]}
            onExit={[Function]}
            onExited={[Function]}
            onExiting={[Function]}
            tag="div"
            timeout={350}
            unmountOnExit={false}
>
            <Styled(Nav)
                        className="ml-auto"
                        navbar={true}
            >
                        <Styled(NavItem)>
                                    <Styled(NavLink)
                                                href="/components/"
                                    >
                                                Premium
                                    </Styled(NavLink)>
                        </Styled(NavItem)>
                        <Styled(NavItem)>
                                    <Styled(NavLink)
                                                href="/components/"
                                    >
                                                Help
                                    </Styled(NavLink)>
                        </Styled(NavItem)>
                        <Styled(NavItem)>
                                    <Styled(NavLink)
                                                href="/components/"
                                    >
                                                Download
                                    </Styled(NavLink)>
                        </Styled(NavItem)>
                        <Styled(NavItem)>
                                    <Styled(NavLink)
                                                href="/components/"
                                    >
                                                Sign up
                                    </Styled(NavLink)>
                        </Styled(NavItem)>
                        <Styled(NavItem)>
                                    <Styled(NavLink)
                                                href="https://github.com/reactstrap/reactstrap"
                                    >
                                                Log In
                                    </Styled(NavLink)>
                        </Styled(NavItem)>
            </Styled(Nav)>
</Collapse>,
        ],
        "expand": "md",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": "Spotify",
            "href": "/",
          },
          "ref": null,
          "rendered": "Spotify",
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "onClick": [Function],
            "tag": "button",
            "type": "button",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "appear": false,
            "children": <Styled(Nav)
              className="ml-auto"
              navbar={true}
>
              <Styled(NavItem)>
                            <Styled(NavLink)
                                          href="/components/"
                            >
                                          Premium
                            </Styled(NavLink)>
              </Styled(NavItem)>
              <Styled(NavItem)>
                            <Styled(NavLink)
                                          href="/components/"
                            >
                                          Help
                            </Styled(NavLink)>
              </Styled(NavItem)>
              <Styled(NavItem)>
                            <Styled(NavLink)
                                          href="/components/"
                            >
                                          Download
                            </Styled(NavLink)>
              </Styled(NavItem)>
              <Styled(NavItem)>
                            <Styled(NavLink)
                                          href="/components/"
                            >
                                          Sign up
                            </Styled(NavLink)>
              </Styled(NavItem)>
              <Styled(NavItem)>
                            <Styled(NavLink)
                                          href="https://github.com/reactstrap/reactstrap"
                            >
                                          Log In
                            </Styled(NavLink)>
              </Styled(NavItem)>
</Styled(Nav)>,
            "enter": true,
            "exit": true,
            "in": false,
            "isOpen": false,
            "mountOnEnter": false,
            "navbar": true,
            "onEnter": [Function],
            "onEntered": [Function],
            "onEntering": [Function],
            "onExit": [Function],
            "onExited": [Function],
            "onExiting": [Function],
            "tag": "div",
            "timeout": 350,
            "unmountOnExit": false,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": Array [
                <Styled(NavItem)>
                  <Styled(NavLink)
                                    href="/components/"
                  >
                                    Premium
                  </Styled(NavLink)>
</Styled(NavItem)>,
                <Styled(NavItem)>
                  <Styled(NavLink)
                                    href="/components/"
                  >
                                    Help
                  </Styled(NavLink)>
</Styled(NavItem)>,
                <Styled(NavItem)>
                  <Styled(NavLink)
                                    href="/components/"
                  >
                                    Download
                  </Styled(NavLink)>
</Styled(NavItem)>,
                <Styled(NavItem)>
                  <Styled(NavLink)
                                    href="/components/"
                  >
                                    Sign up
                  </Styled(NavLink)>
</Styled(NavItem)>,
                <Styled(NavItem)>
                  <Styled(NavLink)
                                    href="https://github.com/reactstrap/reactstrap"
                  >
                                    Log In
                  </Styled(NavLink)>
</Styled(NavItem)>,
              ],
              "className": "ml-auto",
              "navbar": true,
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": <Styled(NavLink)
                    href="/components/"
>
                    Premium
</Styled(NavLink)>,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": "Premium",
                    "href": "/components/",
                  },
                  "ref": null,
                  "rendered": "Premium",
                  "type": [Function],
                },
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": <Styled(NavLink)
                    href="/components/"
>
                    Help
</Styled(NavLink)>,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": "Help",
                    "href": "/components/",
                  },
                  "ref": null,
                  "rendered": "Help",
                  "type": [Function],
                },
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": <Styled(NavLink)
                    href="/components/"
>
                    Download
</Styled(NavLink)>,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": "Download",
                    "href": "/components/",
                  },
                  "ref": null,
                  "rendered": "Download",
                  "type": [Function],
                },
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": <Styled(NavLink)
                    href="/components/"
>
                    Sign up
</Styled(NavLink)>,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": "Sign up",
                    "href": "/components/",
                  },
                  "ref": null,
                  "rendered": "Sign up",
                  "type": [Function],
                },
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": <Styled(NavLink)
                    href="https://github.com/reactstrap/reactstrap"
>
                    Log In
</Styled(NavLink)>,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": "Log In",
                    "href": "https://github.com/reactstrap/reactstrap",
                  },
                  "ref": null,
                  "rendered": "Log In",
                  "type": [Function],
                },
                "type": [Function],
              },
            ],
            "type": [Function],
          },
          "type": [Function],
        },
      ],
      "type": [Function],
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <Styled(Navbar)
          expand="md"
>
          <Styled(NavbarBrand)
                    href="/"
          >
                    Spotify
          </Styled(NavbarBrand)>
          <NavbarToggler
                    onClick={[Function]}
                    tag="button"
                    type="button"
          />
          <Collapse
                    appear={false}
                    enter={true}
                    exit={true}
                    in={false}
                    isOpen={false}
                    mountOnEnter={false}
                    navbar={true}
                    onEnter={[Function]}
                    onEntered={[Function]}
                    onEntering={[Function]}
                    onExit={[Function]}
                    onExited={[Function]}
                    onExiting={[Function]}
                    tag="div"
                    timeout={350}
                    unmountOnExit={false}
          >
                    <Styled(Nav)
                              className="ml-auto"
                              navbar={true}
                    >
                              <Styled(NavItem)>
                                        <Styled(NavLink)
                                                  href="/components/"
                                        >
                                                  Premium
                                        </Styled(NavLink)>
                              </Styled(NavItem)>
                              <Styled(NavItem)>
                                        <Styled(NavLink)
                                                  href="/components/"
                                        >
                                                  Help
                                        </Styled(NavLink)>
                              </Styled(NavItem)>
                              <Styled(NavItem)>
                                        <Styled(NavLink)
                                                  href="/components/"
                                        >
                                                  Download
                                        </Styled(NavLink)>
                              </Styled(NavItem)>
                              <Styled(NavItem)>
                                        <Styled(NavLink)
                                                  href="/components/"
                                        >
                                                  Sign up
                                        </Styled(NavLink)>
                              </Styled(NavItem)>
                              <Styled(NavItem)>
                                        <Styled(NavLink)
                                                  href="https://github.com/reactstrap/reactstrap"
                                        >
                                                  Log In
                                        </Styled(NavLink)>
                              </Styled(NavItem)>
                    </Styled(Nav)>
          </Collapse>
</Styled(Navbar)>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            <Styled(NavbarBrand)
              href="/"
>
              Spotify
</Styled(NavbarBrand)>,
            <NavbarToggler
              onClick={[Function]}
              tag="button"
              type="button"
/>,
            <Collapse
              appear={false}
              enter={true}
              exit={true}
              in={false}
              isOpen={false}
              mountOnEnter={false}
              navbar={true}
              onEnter={[Function]}
              onEntered={[Function]}
              onEntering={[Function]}
              onExit={[Function]}
              onExited={[Function]}
              onExiting={[Function]}
              tag="div"
              timeout={350}
              unmountOnExit={false}
>
              <Styled(Nav)
                            className="ml-auto"
                            navbar={true}
              >
                            <Styled(NavItem)>
                                          <Styled(NavLink)
                                                        href="/components/"
                                          >
                                                        Premium
                                          </Styled(NavLink)>
                            </Styled(NavItem)>
                            <Styled(NavItem)>
                                          <Styled(NavLink)
                                                        href="/components/"
                                          >
                                                        Help
                                          </Styled(NavLink)>
                            </Styled(NavItem)>
                            <Styled(NavItem)>
                                          <Styled(NavLink)
                                                        href="/components/"
                                          >
                                                        Download
                                          </Styled(NavLink)>
                            </Styled(NavItem)>
                            <Styled(NavItem)>
                                          <Styled(NavLink)
                                                        href="/components/"
                                          >
                                                        Sign up
                                          </Styled(NavLink)>
                            </Styled(NavItem)>
                            <Styled(NavItem)>
                                          <Styled(NavLink)
                                                        href="https://github.com/reactstrap/reactstrap"
                                          >
                                                        Log In
                                          </Styled(NavLink)>
                            </Styled(NavItem)>
              </Styled(Nav)>
</Collapse>,
          ],
          "expand": "md",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": "Spotify",
              "href": "/",
            },
            "ref": null,
            "rendered": "Spotify",
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "onClick": [Function],
              "tag": "button",
              "type": "button",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "appear": false,
              "children": <Styled(Nav)
                className="ml-auto"
                navbar={true}
>
                <Styled(NavItem)>
                                <Styled(NavLink)
                                                href="/components/"
                                >
                                                Premium
                                </Styled(NavLink)>
                </Styled(NavItem)>
                <Styled(NavItem)>
                                <Styled(NavLink)
                                                href="/components/"
                                >
                                                Help
                                </Styled(NavLink)>
                </Styled(NavItem)>
                <Styled(NavItem)>
                                <Styled(NavLink)
                                                href="/components/"
                                >
                                                Download
                                </Styled(NavLink)>
                </Styled(NavItem)>
                <Styled(NavItem)>
                                <Styled(NavLink)
                                                href="/components/"
                                >
                                                Sign up
                                </Styled(NavLink)>
                </Styled(NavItem)>
                <Styled(NavItem)>
                                <Styled(NavLink)
                                                href="https://github.com/reactstrap/reactstrap"
                                >
                                                Log In
                                </Styled(NavLink)>
                </Styled(NavItem)>
</Styled(Nav)>,
              "enter": true,
              "exit": true,
              "in": false,
              "isOpen": false,
              "mountOnEnter": false,
              "navbar": true,
              "onEnter": [Function],
              "onEntered": [Function],
              "onEntering": [Function],
              "onExit": [Function],
              "onExited": [Function],
              "onExiting": [Function],
              "tag": "div",
              "timeout": 350,
              "unmountOnExit": false,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": Array [
                  <Styled(NavItem)>
                    <Styled(NavLink)
                                        href="/components/"
                    >
                                        Premium
                    </Styled(NavLink)>
</Styled(NavItem)>,
                  <Styled(NavItem)>
                    <Styled(NavLink)
                                        href="/components/"
                    >
                                        Help
                    </Styled(NavLink)>
</Styled(NavItem)>,
                  <Styled(NavItem)>
                    <Styled(NavLink)
                                        href="/components/"
                    >
                                        Download
                    </Styled(NavLink)>
</Styled(NavItem)>,
                  <Styled(NavItem)>
                    <Styled(NavLink)
                                        href="/components/"
                    >
                                        Sign up
                    </Styled(NavLink)>
</Styled(NavItem)>,
                  <Styled(NavItem)>
                    <Styled(NavLink)
                                        href="https://github.com/reactstrap/reactstrap"
                    >
                                        Log In
                    </Styled(NavLink)>
</Styled(NavItem)>,
                ],
                "className": "ml-auto",
                "navbar": true,
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": <Styled(NavLink)
                      href="/components/"
>
                      Premium
</Styled(NavLink)>,
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "children": "Premium",
                      "href": "/components/",
                    },
                    "ref": null,
                    "rendered": "Premium",
                    "type": [Function],
                  },
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": <Styled(NavLink)
                      href="/components/"
>
                      Help
</Styled(NavLink)>,
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "children": "Help",
                      "href": "/components/",
                    },
                    "ref": null,
                    "rendered": "Help",
                    "type": [Function],
                  },
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": <Styled(NavLink)
                      href="/components/"
>
                      Download
</Styled(NavLink)>,
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "children": "Download",
                      "href": "/components/",
                    },
                    "ref": null,
                    "rendered": "Download",
                    "type": [Function],
                  },
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": <Styled(NavLink)
                      href="/components/"
>
                      Sign up
</Styled(NavLink)>,
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "children": "Sign up",
                      "href": "/components/",
                    },
                    "ref": null,
                    "rendered": "Sign up",
                    "type": [Function],
                  },
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": <Styled(NavLink)
                      href="https://github.com/reactstrap/reactstrap"
>
                      Log In
</Styled(NavLink)>,
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "children": "Log In",
                      "href": "https://github.com/reactstrap/reactstrap",
                    },
                    "ref": null,
                    "rendered": "Log In",
                    "type": [Function],
                  },
                  "type": [Function],
                },
              ],
              "type": [Function],
            },
            "type": [Function],
          },
        ],
        "type": [Function],
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
